services/navigation/data/nav_cfg.json

```diff
@@ -45,8 +45,8 @@
                     }
                 }],
                 "location":{
-                    "latitude":1,
```

Comment:
I have a mild preference for keeping the fake, for-testing-only location being obviously not "real", but up to you whether to revert or not.

Reply:
i can revert :) no biggie